DESCRIPTION

```diff
@@ -29,10 +29,10 @@ Imports:
     rversions,
     stats,
     git2r (>= 0.10.1)
-Suggests: 
+Suggests:
     testthat (>= 0.7),
     BiocInstaller,
-    Rcpp (>= 0.10.0),
+    Rcpp (>= 0.11.0),
```

Comment:
I cannot *promise* that v0.11.0 is necessary, though suspect it is. I will check with v0.10.0.